pinot-minion/src/main/java/com/linkedin/pinot/minion/events/MinionEventNotifier.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.minion.events;
+
+import com.linkedin.pinot.common.config.TableConfig;
+import com.linkedin.pinot.common.data.Schema;
+import com.linkedin.pinot.minion.executor.SegmentConversionInfo;
+
+
+public interface MinionEventNotifier {
+
+  void notifyOnPurge(SegmentConversionInfo segmentConversionInfo);
```

Comment:
It would be more useful to have an interface that is something like `notifyMinionTaskStart/notifyMinionTaskEnd` as opposed to something very specific to the purge task.